addressing_address_map.go

```diff
@@ -0,0 +1,202 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// AddressMap contains information about an address map.
+type AddressMap struct {
+	ID           string                 `json:"id"`
+	Description  string                 `json:"description"`
```

Comment:
```suggestion
	Description  *string                 `json:"description"`
```
Sorry that the docs are not explicit about this (we need to fix it), but this field can be null.